content/2016.12.how-to-develop-a-webapp-with-Kinto.md

```diff
@@ -0,0 +1,235 @@
+Title: How to develop a WebApp with Kinto
+Date: 2016-12-01 10:20
+Slug: develop-a-chat-webapp-with-kinto
+Authors: Enguerran Colson
+Summary: An example of how Kinto handles a webapp's data storage.
+
+## Our goal
+
+Hi folks, today, we are gonna create a simple WebApp that allows anyone to chat in a room accessible through a URL.
+
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_4340C716162DF03B2B97575BFA438C542185694D96D86DB6EB44B989957A3ADE_1480537293391_file.png)
+{.image}
+
+* A really simple chat webapp with minimal UI.
+* We will need a bit of JavaScript to enhance and animate interactions within the application and we will also need a way to store and share the app's data. Our front-end should be able to consume an API that would look like `https://api.discute.re/v1/rooms/4815162342/messages` with [discute.re](http://discute.re) as the domain of our application (*discutere* means to chat in Latin).
```

Comment:
It's confusing to mention a Web *API* with distinct urls than Kinto's. The main purpose of Kinto is to provide this API! 

I get the idea that this example is made to expose the initial issue we want to solve, but this paragraph would need rephrasing so it's more obvious to the reader, eg:

> Ideally, we'd like our application to consume a REST API exposing endpoints like `https://api.discute.re/v1/rooms/12345/messages`, where [discute.re](http://discute.re) is the domain of our application (*discutere* means to chat in Latin).